libtiledbsoma/src/utils/arrow_adapter.cc

```diff
@@ -976,8 +980,8 @@ void ArrowAdapter::_set_current_domain_slot(
             LOG_DEBUG(std::format(
                 "[ArrowAdapter] {} current_domain float {} to {}",
                 name,
-                std::to_string(lo),
```

Comment:
I do seem to recall these were about crash avoidance on _some_ platform (I don't recall which). I'd rather leave these as-is please.